packages/backend/src/core/tx-count/RpcTransactionUpdater.ts

```diff
@@ -0,0 +1,78 @@
+import { Logger, TaskQueue } from '@l2beat/common'
```

Comment:
This class alongside its test counterpart is already on the master branch but in a different directory. Remove them from the PR